### DISCUSSION TITLE
Entry arguments.proto is a duplicate but no duplicate handling strategy

### DISCUSSION THREAD

**Discussion by dhruvpal05** [oppia/oppia-android#5146] [Q&A (Installation)] [answered]
### Operating System

Windows

### Android Studio Version

Android Studio Giraffe | 2022.3.1 Patch 1

### RAM size

16

### Free disk space

850 gb

### Which step are you stuck on?

Not able to run on app in android studio

### Error Log

```bash
Caused by: org.gradle.api.InvalidUserCodeException: Entry arguments.proto is a duplicate but no duplicate handling strategy has been set. Please refer to https://docs.gradle.org/7.2/dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:duplicatesStrategy for details.
```

### Approaches already used to resolve the issue

As mentioned in [#5091](https://github.com/oppia/oppia-android/discussions/5091) I have tried downgrading jdk to 11. Still it did not worked for me.

**Comment by adhiamboperes** [comment]
You may have trouble building Oppia Android on newer versions of android studio because of gradle version incompatibility.

Here are some options to try:
1. Clean and rebuild project.
2. Use android studio bumblebee with java 11
3. Try one of these solutions: https://stackoverflow.com/questions/67265308/gradle-entry-classpath-is-a-duplicate-but-no-duplicate-handling-strategy-has-b

**Comment by nikitaevg** [comment, marked as answer]
I faced the same problem.

Try not using gradle 7.2, but use 6.8. If you face the problem like "Unsupported java Your build is currently configured to use..." with 6.8, you should change gradle's SDK as per [this](https://stackoverflow.com/questions/76177599/android-your-build-is-currently-configured-to-use-java-17-0-6-and-gradle-6-1-1) instruction. Set it to 11.

Hope this helps.

**Reply by vrundraval24** [reply]
Thanks, man! It worked. 😃

**Reply by nikitaevg** [reply]
Glad that helped!

Though you might still want to downgrade to Bumblee. I was unable to launch JUnit tests from the sharedTest directory on the latest Android Studio. I had an error like in the 6th item [here](https://github.com/oppia/oppia-android/blob/08981272a8136b8da0be7c4197480790c36bf100/wiki/Troubleshooting-Installation.md#general-issues).
